src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -625,6 +642,11 @@ public PreferencesViewModel(DynamoViewModel dynamoViewModel)
             SavedChangesTooltip = string.Empty;
 
             this.PropertyChanged += Model_PropertyChanged;
+           
+            preferencesTabs = new List<TabSettings>();
+            preferencesTabs.Add(new TabSettings() { Name = "Features", ExpanderActive = string.Empty });
```

Comment:
Maybe let's add `General` here so the index stable for a while :)

Reply:
@QilongTang fixed, see commit: https://github.com/DynamoDS/Dynamo/pull/11689/commits/4540d66f8358210f8468a3f2ce76482368424622